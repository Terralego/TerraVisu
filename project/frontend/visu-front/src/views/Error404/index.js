import React from 'react';
import { Link } from 'react-router-dom';
import { H1 } from '@blueprintjs/core';

const Error404View = () => (
  <div>
    <H1>Erreur 404 : Page introuvable.</H1>
    <p><strong>Retourner Ã  la page d&#39;accueil: </strong>
      <Link to="/">Accueil</Link>
    </p>
  </div>
);

export default Error404View;
