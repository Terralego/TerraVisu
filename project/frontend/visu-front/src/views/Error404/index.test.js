import React from 'react';
import renderer from 'react-test-renderer';
import Error404 from '.';

jest.mock('react-router-dom', () => ({
  Link: () => null,
}));

jest.mock('@blueprintjs/core', () => ({
  H1: () => <div>blueprint h1</div>,
}));

it('should render correctly', () => {
  const tree = renderer.create(<Error404 />).toJSON();
  expect(tree).toMatchSnapshot();
});
