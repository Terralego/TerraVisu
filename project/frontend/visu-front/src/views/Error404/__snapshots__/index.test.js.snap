// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div>
    blueprint h1
  </div>
  <p>
    <strong>
      Retourner Ã  la page d'accueil: 
    </strong>
  </p>
</div>
`;
